cvat/apps/dashboard/static/dashboard/js/dashboard.js

```diff
@@ -538,29 +538,62 @@ function uploadAnnotationRequest() {
                     }
                     catch(error) {
                         overlay.remove();
-                        showMessage("Parsing errors was occured. " + error);
+                        showMessage("Parsing errors was occurred. " + error);
                         return;
                     }
 
-                    const exportData = createExportContainer();
-                    exportData.create = parsed;
-
                     let asyncSave = function() {
                         $.ajax({
-                            url: '/save/annotation/task/' + window.cvat.dashboard.taskID,
+                            url: '/delete/annotation/task/' + window.cvat.dashboard.taskID,
                             type: 'POST',
```

Comment:
I think method DELETE is better for delete request

Reply:
OK